internals/data_types.py

```diff
@@ -263,3 +264,31 @@ class VerboseFeatureDict(TypedDict):
   is_enterprise_feature: bool
   updated_display: str | None
   new_crbug_url: str | None
+
+
+@dataclass
+class OriginTrialInfo():
+  def __init__(self, api_trial):
+    self.id = api_trial.get('id', '')
+    self.ot_type = api_trial.get('type', '')
+    self.display_name = api_trial.get('displayName', '')
+    self.description = api_trial.get('description', '')
+    self.origin_trial_feature_name = api_trial.get('originTrialFeatureName', '')
```

Comment:
Will the frontend need to know about the `originTrialFeatureName` field?

Reply:
Wouldn't we want to use the `id` field for that purpose instead?

Reply:
Yes, but we cannot know the OT ID since we do not have access to that until trial creation is handled on Chromestatus, or we ask feature owners to supply the OT ID in Chromestatus. We ask feature owners to supply this feature name field, and with this + the chromestatus URL, we can begin to associate origin trials to their respective feature stage in Chromestatus.

This is not an implementation we want to use permanently, but it would make it possible with what we currently have to link OTs with their feature, and then when trial creation is handled in Chromestatus, we can save the trial ID directly to make this association.

I am willing to talk more about this if you would like to talk about the efficacy of this approach 🙂 I won't deny that it's half-baked 😅

Reply:
No worries! Thanks for the explanation. It helped. I get what you're going for now. Let's go with what you have and we can revisit in the future if needed.